compiler/main/runpasses.cpp

```diff
@@ -40,49 +40,49 @@ struct PassInfo {
+#define LOG_flattenClasses                     LOG_NO_SHORT
 #define LOG_normalize                          'N'
-#define LOG_checkNormalized                    NUL
+#define LOG_checkNormalized                    LOG_NEVER
 #define LOG_buildDefaultFunctions              'D'
 #define LOG_createTaskFunctions                't'
-#define LOG_expandExternArrayCalls             NUL
+#define LOG_expandExternArrayCalls             LOG_NO_SHORT
 #define LOG_resolve                            'R'
 #define LOG_resolveIntents                     'i'
-#define LOG_checkResolved                      NUL
-#define LOG_replaceArrayAccessesWithRefTemps   'T'
-#define LOG_processIteratorYields              'y'
-#define LOG_flattenFunctions                   'e'
-#define LOG_cullOverReferences                 'O'
-#define LOG_lowerErrorHandling                 NUL
+#define LOG_checkResolved                      LOG_NEVER
+#define LOG_replaceArrayAccessesWithRefTemps   LOG_NO_SHORT
+#define LOG_processIteratorYields              LOG_NO_SHORT
+#define LOG_flattenFunctions                   LOG_NO_SHORT
+#define LOG_cullOverReferences                 LOG_NO_SHORT
+#define LOG_lowerErrorHandling                 LOG_NO_SHORT
 #define LOG_callDestructors                    'd'
 #define LOG_lowerIterators                     'L'
 #define LOG_parallel                           'P'
-#define LOG_prune                              'X'
-#define LOG_bulkCopyRecords                    'B'
-#define LOG_removeUnnecessaryAutoCopyCalls     'U'
+#define LOG_prune                              LOG_NO_SHORT
+#define LOG_bulkCopyRecords                    LOG_NO_SHORT
+#define LOG_removeUnnecessaryAutoCopyCalls     LOG_NO_SHORT
 #define LOG_inlineFunctions                    'I'
-#define LOG_scalarReplace                      'r'
-#define LOG_refPropagation                     'g'
-#define LOG_copyPropagation                    'G'
-#define LOG_deadCodeElimination                'x'
-#define LOG_removeWrapRecords                  'w'
-#define LOG_removeEmptyRecords                 'm'
-#define LOG_localizeGlobals                    'l'
-#define LOG_loopInvariantCodeMotion            'q'
-#define LOG_prune2                             'Y'
-#define LOG_returnStarTuplesByRefArgs          's'
+#define LOG_scalarReplace                      LOG_NO_SHORT
+#define LOG_refPropagation                     LOG_NO_SHORT
+#define LOG_copyPropagation                    LOG_NO_SHORT
+#define LOG_deadCodeElimination                LOG_NO_SHORT
+#define LOG_removeWrapRecords                  LOG_NO_SHORT
+#define LOG_removeEmptyRecords                 LOG_NO_SHORT
+#define LOG_localizeGlobals                    LOG_NO_SHORT
+#define LOG_loopInvariantCodeMotion            LOG_NO_SHORT
+#define LOG_prune2                             LOG_NO_SHORT
+#define LOG_returnStarTuplesByRefArgs          LOG_NO_SHORT
 #define LOG_insertWideReferences               'W'
 #define LOG_optimizeOnClauses                  'o'
```

Comment:
I'm only leaving 1-character names for "the big ones":
 * parse
 * scope resolve
 * normalize
 * resolve
 * codegen

It's likely somebody else would create a different set but this particular set allows each 1-letter abbreviation to be just the 1st letter.